test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -23,6 +23,14 @@ namespace Dotnet.Integration.Test
     public class PackCommandTests
     {
         private DotnetIntegrationTestFixture msbuildFixture;
+        // Specifies a target framework for projects used during testing.  This should match the framework that this project was built against.
+#if NET8_0
+        private const string ProjectTargetFramework = "net8.0";
+#elif NET9_0
+        private const string ProjectTargetFramework = "net9.0";
+#else
+        Update the logic for which target framework to use for tests projects!!!
```

Comment:
```suggestion
        #error Update the logic for which target framework to use for tests projects!!!
```

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#error-and-warning-information